iothub/service/tests/ServiceClientTests.cs

```diff
@@ -12,83 +12,14 @@ namespace Microsoft.Azure.Devices.Api.Test
     using Microsoft.Azure.Amqp;
     using Microsoft.Azure.Devices;
     using Microsoft.Azure.Devices.Common.Exceptions;
+    using Microsoft.Azure.Devices.Http2;
```

Comment:
Probably unused

Reply:
@timtay-microsoft I have added the suggested changes. Please have a look.